.github/workflows/conformance-kubespray.yaml

```diff
@@ -0,0 +1,207 @@
+name: Conformance Kubespray (ci-kubespray)
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  workflow_dispatch:
+    inputs:
+      PR-number:
+        description: "Pull request number."
+        required: true
+      context-ref:
+        description: "Context in which the workflow runs. If PR is from a fork, will be the PR target branch (general case). If PR is NOT from a fork, will be the PR branch itself (this allows committers to test changes to workflows directly from PRs)."
+        required: true
+      SHA:
+        description: "SHA under test (head of the PR branch)."
+        required: true
+      extra-args:
+        description: "[JSON object] Arbitrary arguments passed from the trigger comment via regex capture group. Parse with 'fromJson(inputs.extra-args).argName' in workflow."
+        required: false
+        default: '{}'
+  push:
+    branches:
+      - 'main'
+      - 'renovate/main-**'
+  schedule:
+    - cron:  '0 0/12 * * *'
```

Comment:
I don't think we need to run this so frequently. Maybe once before release or prerelease?

```suggestion
```

Reply:
Probably the big remaining question is: Who will look at the results and when? We could defer to you if you're willing to periodically check the results and file issues/fixes if it breaks. Or we could extend the [RC checklist](https://github.com/cilium/release/blob/main/.github/ISSUE_TEMPLATE/release_template_rc_branch.md).

Reply:
I'm happy to periodically check and maintain the results of this kubespray workflow. 
I believe other kubespray community members like @cyclinder and @yankay would also be willing to help with the maintenance.